pdk/1.0/modules/ROOT/pages/policies-pdk-debug-deployed-policies.adoc

```diff
@@ -0,0 +1,34 @@
+= Debugging Deployed Custom Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+If a custom policy deployed on a Flex Gateway is malfunctioning, there are some different methods to debug the policy:
+
+* <<debug-environment>>
+* <<integration-testing>>
+* <<debug-logs>>
+
+[[debug-playground]]
+== Use PDK Debugging Playground
```

Comment:
```suggestion
== Use the PDK Debugging Playground
```
How does marketing want the debugging playground positioned? (With a "the" or without?)